app/Models/Entry.php

```diff
@@ -68,11 +69,15 @@ public function date($raw = false) {
 			return timestamptodate($this->date);
 		}
 	}
-	public function dateAdded($raw = false) {
-		$date = intval(substr($this->id, 0, -6));
+	public function dateAdded($raw = false, $microsecond = false) {
 		if ($raw) {
-			return $date;
+			if ($microsecond) {
+				return $this->date_added;
+			} else {
+				return intval(substr($this->id, 0, -6));
```

Comment:
I realize it already said this, but it still kind of tickles.

Reply:
"kind of tickles" ? 🤔

By the way, whith your comment, I note I'm still using `$this->id` while I wanted to use `$this->date_added`, when not `$microsecond`, for coherency / homogeneity of this getter 😩🤦‍♂️

In practice, it doesn't change anything (we are never using a changed `date_added` when `$microsecond == false`), but can be good to fix it, no ? Do I open a little PR for that ?

Reply:
Yes, please fix it. :+1: 

It kind of tickles because it's just chomping off part of the string rather than rounding. I doubt it matters, but it feels funny. ;-)

Reply:
https://github.com/FreshRSS/FreshRSS/pull/2774